### ISSUE TITLE
A11Y - Zoom doesn't work properly

### ISSUE THREAD

**Issue by motching** [WOEIP/truck-tracker#41]
Probably it's how we define font-sizes, etc

**Comment by theecrit** [comment]
@motching My understanding is that font sizing should be done via REM, with 1rem set to default browser setting (which allows for native browser zooming).
